WcaOnRails/config/locales/ja.yml

```
@@ -40,6 +40,8 @@ ja:
     SZ: エスワティニ
     #original_hash: 4d206a8
     TT: トリニダード・トバゴ
+    #original_hash: fefc818
+    TW: 中華台北
```

```suggestion
    TW: チャイニーズタイペイ
```
This is how Japanese use in sports broadcast.